index.js

```diff
@@ -2,4 +2,26 @@ console.clear();
 
 //1.
 
+const bookmark = document.querySelector('[data-js="bookmark-button"]');
+
+bookmark.addEventListener("click", () => {
+  bookmark.classList.toggle("question-card__bookmark--remove");
+});
+
+const answerButton = document.querySelector('[data-js="buttonContainer"]');
```

Comment:
You could think about removing the button container (in general) and just work with the button. Would work the same way. 😊